### PR TITLE
Fixed link.

### PR DESCRIPTION
**Pull request by valeonte**
Old link was pointing to a location that has moved.

### CODE REVIEW

**Comment by PRMerger6** [comment]
@valeonte : Thanks for your contribution to the Azure documentation! The author, @markingmyname, has been notified to review your proposed change.

**Comment by wsmelton** [comment]
@valeonte this link needs to be pointed back to the docs version of the page and not the old format.

The page you want is found [here](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/reporting-services/report-data/data-connections-data-sources-and-connection-strings-report-builder-and-ssrs.md).

You would reference the link to this page in this manner: `../../reporting-services/report-data/data-connections-data-sources-and-connection-strings-report-builder-and-ssrs.md`

**Comment by craigg-msft** [comment]
@markingmyname please review.

**Comment by PRMerger9** [comment]
@valeonte : Thanks for your contribution to the Azure documentation! The author, @markingmyname, has been notified to review your proposed change.

**Comment by valeonte** [comment]
Followed suggestion for @wsmelton and linked properly. Thanks @wsmelton!

**Comment by craigg-msft** [comment]
@markingmyname  please review.

File: docs/reporting-services/report-data/xml-connection-type-ssrs.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ ms.workload: "On Demand"
 |XML document|`http://localhost/XML/Customers.xml`|  
 |Embedded XML document|*Empty*|  
   
- For more connection string examples, see [Data Connections, Data Sources, and Connection Strings in Report Builder](http://msdn.microsoft.com/library/7e103637-4371-43d7-821c-d269c2cc1b34).  
+ For more connection string examples, see [Data Connections, Data Sources, and Connection Strings in Report Builder](../../reporting-services/report-data/data-connections-data-sources-and-connection-strings-report-builder-and-ssrs.md).
```
Comment (markingmyname) [inline docs/reporting-services/report-data/xml-connection-type-ssrs.md]: i see that the change is not a URL, but a pointer to a different file.  I did check the original URL and that is no longer a useful link.  If the new pointer is directing to the samples, then I have no issue with the change.  Was not able to check that myself though.

**Comment by craigg-msft** [comment]
Thank you @valeonte  and @markingmyname  merging.

**Pull request merged**